library/Icingadb/Widget/ItemList/StateListItem.php

```diff
@@ -28,6 +28,9 @@ abstract class StateListItem extends BaseListItem
     /** @var State The state of the item */
     protected $state;
 
+    /** @var string Default path of icon image */
+    const DEFAULT_ICON_PATH = 'img/icons/';
```

Comment:
Use constants with care. Is it a value specific to this widget? No, it is specific to Icinga Web 2.

So we have a publicly accessible constant here, that is related to the framework, not even the module.
Anyone using the constant from the outside will do so only if using a `StateListItem` anyway for some reason. (hopefully) But there's literally zero reason why such users should then require it to begin with. Icon images are already supported.

tl;dr, Don't use a constant for this. This constant would fit in the `Environment` class of Icinga Web 2 for example, but we're in the module here so a hard coded string suffices.

Reply:
I see, thanks for explanation. :)